jax_unirep/sampler.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+
+
+def check_acceptance(best: float, candidate: float):
+    """
+    This function checks whether we want to 
```

Comment:
Summary line needed.

```suggestion
    Return boolean decision on whether the candidate mutant is accepted or not.
    
    This function checks whether we want to 
```